### ISSUE TITLE
[instrument list] 2 parameter mode doesn't work in the timepoint list

### ISSUE THREAD

**Issue by lingma** [aces/Loris#6323]
**Describe the bug**
If you read the code, you can see that the program seems to support 2 parameter mode in timepoint list, but in fact this doesn't work. I am not sure whether this feature is necessary to check.

![image](https://user-images.githubusercontent.com/5295785/78621412-84443200-7850-11ea-89a4-dc466b939cc6.png)

**To Reproduce**
Steps to reproduce the behavior (attach screenshots if applicable):
1. Go to 'timepoint list
2. put the session id at the end of the URL to access directly
3. See error

**What did you expect to happen?**
If the program says supported, then do it and add to the test cases, otherwise remove that part of the code.

**Server Environment (if known):**
- LORIS Version: 23.0

**Comment by driusan** [comment]
Is this linked to from anywhere? If not maybe we should just remove the feature.

**Comment by lingma** [comment]
This one is not working at least from 20.0, it is supposed to be to support .htaccess url rewrite, do we still need this feature? My fix could make it work, but if this feature is not necessary, we don't need further spend time to create test case

**Comment by lingma** [comment]
> Is this linked to from anywhere? If not maybe we should just remove the feature.

I don't think this feature is used any where, otherwise someone should have reported as a bug

**Comment by lingma** [comment]
I have removed this two parameter access from the router. I checked, found that the program was not correct to process 2 parameter mode since the version V20.1, while the V20.0 should be correct.

We can use the .htaccess to rewrite URL instead of processing by ourselves. I don't see the necessity and the test will be simple without this two parameter access mode support.

I did not close this issue for now for reference purpose.
